packages/patternfly-4/react-charts/src/components/ChartLabel/__snapshots__/ChartLabel.test.tsx.snap

```diff
@@ -7,7 +7,9 @@ exports[`Chart 1`] = `
   lineHeight={1}
   style={
     Object {
-      "fontFamily": "overpass, overpass, open sans, -apple-system, blinkmacsystemfont, Segoe UI, roboto, Helvetica Neue, arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol",
+      "fontFamily": "var(--pf-chart-global--FontFamily)",
+      "fontSize": 14,
```

Comment:
we have vars for the font sizes used in charts - this one is `--pf-chart-global--FontSize--sm`

https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/_chart-globals.scss#L55-L61

Reply:
Victory typically requires raw values for colors, pixels, etc. The font size; for example, must be output as a value because Victory concatenates that with 'px'. 

The font family and letterSpacing are one of the few places where we can output a variable.

Reply:
Should we remove the unit from these vars so they're usable? We do that with other properties like padding, margin, width, height, etc.

Reply:
There is no unit in our pf-core chart vars, it's just a raw number.

Victory is appending the unit automatically. Thus, a CSS variable breaks for properties like `fontSize`, `padding`, etc.

I've created an issue with Victory, asking they change that.
`https://github.com/FormidableLabs/victory/issues/1357`

Reply:
There is no unit in our pf-core chart vars for this very reason, it's just a raw number.

Victory is appending the unit automatically. Thus, a CSS variable breaks for properties like `fontSize`, `padding`, etc.

I've created an issue with Victory, asking they change that. Although, I suspect that won't be possible because they use the raw values to calculate heights & widths of labels, legends, etc.
`https://github.com/FormidableLabs/victory/issues/1357`